### PR TITLE
Fix bugs/questionnaire page

### PR DESCRIPTION
**Pull request by lustigy96**

### CODE REVIEW

File: src/common/DeletionDialog/DeletionDialog.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      ? (
+        <CustomDialog
+          open={open}
+          paperClassName={classes.root}
+          onClose={onClose}
+          title=''
+          content={content}
+        />
+      )
+      : <></>
```
Comment (carmeltsadok) [inline src/common/DeletionDialog/DeletionDialog.jsx:41]: no need for this condition when there is an `open={open}` prop to the dialog

File: src/pages/QuestionnaireSchemaPage/components/LinearScaleQuestion/LinearScaleQuestion.jsx [hunk 2]
```diff
@@ -31,17 +37,25 @@ const LinearScaleQuestion = ({ linearScale, setLinearScale }) => {
     setLinearScale(tmpScale);
   };
 
+  useEffect(() => {
+    setMinMaxTitleError(linearScale.minTitle.length === 0 || linearScale.maxTitle.length === 0);
```
Comment (carmeltsadok) [inline src/pages/QuestionnaireSchemaPage/components/LinearScaleQuestion/LinearScaleQuestion.jsx:41]:
```suggestion
    setMinMaxTitleError(!linearScale.minTitle.length || !linearScale.maxTitle.length);
```

File: src/pages/QuestionnaireSchemaPage/components/LinearScaleQuestion/LinearScaleQuestion.styles.js [hunk 3]
```diff
@@ -20,7 +20,12 @@ const useStyles = makeStyles((theme) => ({
     boxSizing: 'border-box',
     marginTop: '0.1rem',
     maxWidth: '7rem',
+    '&$inputError': {
+      color: 'red',
+      borderColor: 'red',
+    },
   },
+  inputError: {},
```
Comment (carmeltsadok) [inline src/pages/QuestionnaireSchemaPage/components/LinearScaleQuestion/LinearScaleQuestion.styles.js:28]:
```suggestion
```

Comment (carmeltsadok) [reply src/pages/QuestionnaireSchemaPage/components/LinearScaleQuestion/LinearScaleQuestion.styles.js:28]: remove class from the component that used it too

File: src/pages/QuestionnaireSchemaPage/components/MultipleChoice/MultipleChoice.styles.js [hunk 4]
```diff
@@ -19,7 +19,13 @@ const useStyles = makeStyles((theme) => ({
     width: '100%',
     padding: '0.1rem 1rem',
     marginBottom: '0.3rem',
+    boxSizing: 'border-box',
+    '&$inputError': {
+      color: 'red',
+      borderColor: 'red',
+    },
   },
+  inputError: {},
```
Comment (carmeltsadok) [inline src/pages/QuestionnaireSchemaPage/components/MultipleChoice/MultipleChoice.styles.js:28]: too..

File: src/pages/QuestionnaireSchemaPage/components/Question/Question.jsx [hunk 5]
```diff
@@ -174,19 +208,18 @@ const Question = ({
...
+          classes={{ root: classes.input, error: classes.erroredInput }}
+          disableUnderline
           margin='dense'
-          id='question-title'
           type='text'
           variant='standard'
-          error={error}
-          placeholder={error ? '*' : ''}
+          error={showErrors && titleError}
+          // placeholder={(showErrors && titleError) ? '*' : ''}
```
Comment (carmeltsadok) [inline src/pages/QuestionnaireSchemaPage/components/Question/Question.jsx:218]:
```suggestion
```

File: src/pages/QuestionnaireSchemaPage/components/Question/Question.styles.js [hunk 6]
```diff
@@ -42,15 +49,28 @@ const useStyles = makeStyles((theme) => ({
...
-      fontWeight: '800',
+      borderColor: 'red',
     },
     [theme.breakpoints.down('sm')]: {
       width: '80%',
     },
   },
+  erroredInput: {
+
+  },
```
Comment (carmeltsadok) [inline src/pages/QuestionnaireSchemaPage/components/Question/Question.styles.js:62]: too

File: src/pages/QuestionnaireSchemaPage/components/QuestionnaireDialog/QuestionnaireDialog.styles.js [hunk 7]
```diff
@@ -88,6 +88,13 @@ const useStyles = makeStyles((theme) => ({
...
     padding: '0.6em 1rem',
     boxSizing: 'border-box',
+    '&$erroredInput': {
+      color: 'red',
+      borderColor: 'red',
+    },
+  },
+  erroredInput: {
+
   },
```
Comment (carmeltsadok) [inline src/pages/QuestionnaireSchemaPage/components/QuestionnaireDialog/QuestionnaireDialog.styles.js:98]: too

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: src/pages/QuestionnaireSchemaPage/components/LinearScaleQuestion/LinearScaleQuestion.styles.js (hunk 3 above)
Comment (lustigy96) [reply src/pages/QuestionnaireSchemaPage/components/LinearScaleQuestion/LinearScaleQuestion.styles.js:28]: this is needed and I think it is trivial when you override matirialUI events

File: src/pages/QuestionnaireSchemaPage/components/MultipleChoice/MultipleChoice.styles.js (hunk 4 above)
Comment (lustigy96) [reply src/pages/QuestionnaireSchemaPage/components/MultipleChoice/MultipleChoice.styles.js:28]: this is needed and I think it is trivial when you override matirialUI events

**Review by carmeltsadok — APPROVED** [review]

**Pull request merged**